crates/fluvio-sc/src/init.rs

```diff
@@ -142,6 +143,12 @@ where
                     ctx,
                     Arc::new(BasicAuthorization::new(policy)),
                 ));
+            } else if ctx.config().tls {
+                info!("using spu remote authorization");
+                start_public_server(AuthGlobalContext::new(
+                    ctx,
+                    Arc::new(RemoteAuthorization::new()),
```

Comment:
Hmm.  Can't have two different policy.  Instead this should be unified

Reply:
True, make sense I just make the policy optional for BasicAuthorization?

Reply:
use whatever as default.  no need for change

Reply:
scope should be part of auth context